src/plugins/timelist/Timelist.vue

```diff
@@ -110,7 +110,28 @@ export default {
     },
     mounted() {
         this.isEditing = this.openmct.editor.isEditing();
-        this.timestamp = Date.now();
+        this.timestamp = this.openmct.time.clock() ? this.openmct.time.clock().currentValue : undefined;
+        this.openmct.time.on('clock', (newClock) => {
+            this.filterValue = this.domainObject.configuration.filter;
+            //newclock can be undefined
+            if (newClock === undefined) {
+                // Show everything for fixed time
+                this.hideAll = false;
```

Comment:
It looks like the same logic is being executed below: https://github.com/nasa/openmct/blob/df0489ab69c76ef4175bd424d3222b60bc577321/src/plugins/timelist/Timelist.vue#L156

Let's make a method out of this code block and use it in both places.